dataactcore/config.py

```diff
@@ -77,6 +81,8 @@
     # just use the directory
     broker_files = os.path.split(broker_files)[0]
 normpath(broker_files)
+if broker_files[-1] != os.path.sep:
+    broker_files = "".join([broker_files, os.path.sep])
```

Comment:
Will this config be used at all if we're using aws? If so, the `os.path.sep` won't match what AWS is expecting (which is always a forward slash).

Minor details:
* `str.endswith()` is a little nicer than `[-1]`. It handles empty strings and the hypothetical case where `os.path.sep` is not a single character
* the `"".join(` idiom is a little strange for string concatenation. Particularly if you're just joining two strings, a plus is more common:
```python
broker_files = broker_files + os.path.sep
# or
broker_files += os.path.sep
```